api/v1/controllers/categories.js

```diff
@@ -0,0 +1,53 @@
+import { toInt } from '../helpers/functions';
+
+// Import data structure for categories
+import { allCategories, Category } from '../models/Category';
+
+export default {
+
+  getAllCategories(req, res) {
+    res.status(200).json(allCategories);
+  },
+
+  getCategoryById(req, res) {
+    const categoryId = toInt(req.params.categoryId);
+    if (!categoryId) {
+      res.status(422).send({ errors: { categoryId: 'Category Id is required' } });
```

Comment:
Prefer using 400 for bad requests and not 422 which is for unprocessable entity

Reply:
Noted!